### ISSUE TITLE
document pysam as a requirement and maybe list m1 macbook as a known challenge?

### ISSUE THREAD

**Issue by ctb** [codemeleon/seqPanther#6]
tl;dr pysam is required for install, but is not available as wheel and is tricky to compile from scratch. suggest documenting that pysam is a requirement in the readme.

for my M1 Macbook, pysam is not available as a binary via pypi and must be compiled, but this fails (in the last decade I don't think I've ever gotten pysam to compile, not even on linux 😂)

pysam is available for bioconda, but not for M1 macbooks with the ARM architecture. So I can't install seqpanther on my laptop.

that's ok, I can test it on linux, but I think it might be worth putting in a note that if you can install pysam in your conda environment, life might be easier!

**Comment by jsan4christ** [comment]
@cbrueffer I am using an M1 and its installing without any issue on my computer. Let me know if you still have a problem installing please let me know.

**Comment by cbrueffer** [comment]
Tagging @ctb instead of me :-)
